toqito/matrix_props/is_etf.py

```diff
@@ -0,0 +1,69 @@
+"""Checks if matrix forms an equilangular tight frame (ETF)."""
+
+import numpy as np
+
+from toqito.matrix_ops import vectors_to_gram_matrix
+
+
+def is_etf(mat: np.ndarray, rtol: float = 1e-9, atol: float = 1e-9) -> bool:
+    r"""Check if a matrix constitutes an equilangular tight frame (ETF).
+
+    Definition taken from:
+    :cite:`hoffman2011equilangular`.
+
+    A matrix :math:`A` constitutes an equilangular tight frame if it satisfies three conditions:
+
+    1. Each column of the matrix :math:`A` has unit norm.
+    2. All diagonal elements of the Gram matrix are one, and its off-diagonal elements are constant.
+    3. :math:`AA^* = (ncols/nrows) I`. Here, :math:`A^*` is the conjugate transpose of :math:`A`
+       and :math:`I` is the identity matrix.
+
+    Example
+    ========
+    >>> # Example of a ETF matrix.
+    >>> import numpy as np
+    >>> from toqito.matrix_ops import vectors_to_gram_matrix
+    >>> from toqito.matrix_props import is_etf
+    >>> mat = np.array([[1, -1/2, -1/2], [0, np.sqrt(3)/2, -np.sqrt(3)/2]])
+    >>> is_etf(mat)
+    True
+
+    References
+    ==========
+    .. bibliography::
+        :filter: docname in docnames
+        
+
+    Args:
+        mat (np.ndarray): The matrix to check.
+        rtol: Relative tolerance for numerical comparisons (default: 1e-9).
+        atol: Absolute tolerance for numerical comparisons (default: 1e-9).
+        
+    Returns:
```

Comment:
Please refer to how we format the "returns" part of the docstrings for comparison and consistency in other functions.